store/tikv/lock_resolver.go

```diff
@@ -410,22 +414,21 @@ func (lr *LockResolver) getTxnStatusFromLock(bo *Backoffer, l *Lock, callerStart
 			return TxnStatus{}, err
 		}
 
-		if l.LockType == kvrpcpb.Op_PessimisticLock {
-			return TxnStatus{ttl: l.TTL}, nil
-		}
-
 		// Handle txnNotFound error.
 		// getTxnStatus() returns it when the secondary locks exist while the primary lock doesn't.
 		// This is likely to happen in the concurrently prewrite when secondary regions
 		// success before the primary region.
 		if err := bo.Backoff(boTxnNotFound, err); err != nil {
```

Comment:
The `ManagedLockTTL` is 20s, and `pessimisticLockMaxBackoff` is 10s，the transaction will exceed the max backoff and return error before the TTL reached.